params.go

```diff
@@ -85,3 +95,20 @@ func ParseBrowsersParam(r *http.Request) (browsers []string, err error) {
 	sort.Strings(browsers)
 	return browsers, nil
 }
+
+// ParseMaxCountParam parses the 'max-count' parameter as an integer.
+func ParseMaxCountParam(r *http.Request) (count int, err error) {
+	count = MaxCountDefaultValue
+	if browsersParam := r.URL.Query().Get("max-count"); browsersParam != "" {
```

Comment:
Should this by maxCountParam or countParam instead of browsersParam?

Reply:
Fixed (CopyPasta)